BEP92.md

```diff
@@ -0,0 +1,73 @@
+# BEP-92: Implementation of SHA3-256 FIPS 202 hash and EcRecover Uncompressed public key precompiled contracts
+
+- [BEP-92: Implementation of SHA3-256 FIPS 202 hash precompiled and EcRecover Uncompressed public key precompiled](#bep-92)
+  - [1. Summary](#1-summary)
+  - [2. Abstract](#2-abstract)
+  - [3. Status](#3-status)
+  - [4. Motivation](#4-motivation)
+  - [5. Specification](#5-specification)
+  - [6. Gas Consumption](#6-gas)
+  - [7. Test Data](#7-testdata)
+  - [8. License](#6-license)
+  
+## 1. Summary
+
+This BEP describes a proposal for Implementation of SHA3-256 as precompiled for BSC
+
+## 2. Abstract
+
+BEP-92 Proposal describes a addition of implementation of SHA3-256 FIPS 202 hash and EcRecover Uncompressed public key methods as precompiled native contracts

```

Comment:
`a addition` ==> `an addition`

Reply:
Thanks @simsonraj,  the BEP-92 looks fine to me. No more comments

Reply:
Corrected this, thank you @guagualvcha!